### ISSUE TITLE
RangeError: Maximum call stack size exceeded in case of big request model

### ISSUE THREAD

**Issue by gaurav517** [springfox/springfox#2473]
What version of the library are you using? Is it the latest version? The latest released version is [ ![Download](https://api.bintray.com/packages/springfox/maven-repo/springfox/images/download.svg) ](https://bintray.com/springfox/maven-repo/springfox/_latestVersion)
**2.9.0**

 **Bug report**
In case of big request json spec, I get following in console after trying to expand the operation in swagger-ui and the section of operation shows endless spinner. After refreshing we see the operation details.
If request is not big but only response is big, it works fine so I am inferring that it might be related to the size of request model or the combined size, not sure.

```
system.js:461 RangeError: Maximum call stack size exceeded
    at immutable.js:2405
    at Le.pe.withMutations (immutable.js:1353)
    at He (immutable.js:2405)
    at Le.set (immutable.js:2096)
    at nt (immutable.js:2780)
    at Ze.set (immutable.js:2686)
    at immutable.js:2651
    at immutable.js:3016
    at immutable.js:2699
    at Le.__iterate (immutable.js:2206)
```
Thanks a lot.

**Comment by dilipkrish** [comment]
This is probably a bug in [Swagger-ui](https://github.com/swagger-api/swagger-ui). Its not the size as much as how its handling recursive models.

**Comment by gaurav517** [comment]
I see. The models with issues indeed have layers of recursion. Yea, seems javascript is falling into some endless recursion. Should we open it in Swagger-ui repo? Please let me know after your initial analysis. Thanks.

**Comment by dilipkrish** [comment]
That was the analysis :)

**Comment by gaurav517** [comment]
Created a ticket in swagger-ui repo.
https://github.com/swagger-api/swagger-ui/issues/4637

**Comment by dilipkrish** [comment]
Would be useful to provide them the model IMO, to use as a test.

**Comment by JavaHelper** [comment]
@All - Any updates on this issue ? How we can solve this ?

**Comment by dilipkrish** [comment]
This issue was closed, what version are you using @JavaHelper

**Comment by JavaHelper** [comment]
I am using `<springfox-swagger-version>2.9.2</springfox-swagger-version>`

**Comment by dilipkrish** [comment]
It seems to be a closed issue in swagger-ui

**Comment by JavaHelper** [comment]
Nope, it seems not fixed. I am still getting the same error. Swagger version

```
<springfox-swagger-version>2.9.2</springfox-swagger-version>
<dependency>
			<groupId>io.springfox</groupId>
			<artifactId>springfox-swagger-ui</artifactId>
			<version>${springfox-swagger-version}</version>
		</dependency>
		<dependency>
			<groupId>io.springfox</groupId>
			<artifactId>springfox-swagger2</artifactId>
			<version>${springfox-swagger-version}</version>
			<exclusions>
				<exclusion>
					<groupId>io.swagger</groupId>
					<artifactId>swagger-models</artifactId>
				</exclusion>
			</exclusions>
		</dependency>
```

![image](https://user-images.githubusercontent.com/8208031/62706839-e91f6200-ba0d-11e9-9c62-cc5187925780.png)

**Comment by JavaHelper** [comment]
@All - Any updates on this issue please ?

**Comment by dilipkrish** [comment]
@ghost could you provide a model that reproduces the issue

**Comment by rattada** [comment]
i am using 4.01 version for asp.net core

![image](https://user-images.githubusercontent.com/2851831/63506277-d79c8680-c4f2-11e9-98de-486e474c05b2.png)

This issue is still exist, please provide solution
Thannks in Advance
